Update docs to LDS Full Landonline Data Dictionary 2.6

strk opened the pull request:
See #229

vlindsay commented on doc/lds-full-landonline-data-dictionary-and-models.md:
```diff
@@ -292,16 +295,16 @@ __*AUDIT_ID*__     | INTEGER | Id used to link to the associated audit details.
 ## Alias [(https://data.linz.govt.nz/table/51982)](https://data.linz.govt.nz/table/51982)
 ### Description:
-Individual proprietors may have one or more alternate names or (aliases). This entity stores all of the alternate names used by an individual proprietor. Corporate proprietors can not have aliases.<br>
+Individual registered owners may have one or more alternate names or aliases. This entity stores all of the alternate names used by an individual registered owner. Corporate owners can not have aliases.<br>
```
good

vlindsay commented on doc/lds-full-landonline-data-dictionary-and-models.md:
```diff
@@ -517,7 +520,7 @@ NAME               | VARCHAR(255)           | The name of the encumbrancee. <br>
 ## Estate Share [(https://data.linz.govt.nz/table/52065)](https://data.linz.govt.nz/table/52065)
 ### Description:
-An estate may be owned in shares by proprietors. For example John and Bill may each own a 1/2 share in the land. This table contains one row for each share that exists in an estate. <br>
+An estate may be owned in shares by registered owners. For example John and Bill may each own a 1/2 share in the land. This table contains one row for each share that exists in an estate. <br>
```
There is an additional change here, where I changed the names 'John and Bill' to 'Marian and Anne' - just because I like seeing women's names in docs too:)

vlindsay reviewed (commented):
Looks great. good work on manually finding the majority of changes. I have marked a couple of points below. There are also a few additional changes you did not find. How about you finish your version here, then I will edit an updated one with the last few changes? 
The image of one of the model diagrams (titles) also needs updating I believe.

strk commented:
On Tue, Sep 08, 2020 at 07:19:51PM -0700, Vicki Lindsay wrote:
> There is an additional change here, where I changed the names 'John and Bill' to 'Marian and Anne' - just because I like seeing women's names in docs too:)

Changed with 3e80fa1 :)

strk commented:
On Tue, Sep 08, 2020 at 08:16:13PM -0700, Vicki Lindsay wrote:
> @vlindsay commented on this pull request.
> 
> Looks great. good work on manually finding the majority of changes. I have marked a couple of points below. There are also a few additional changes you did not find. How about you finish your version here, then I will edit an updated one with the last few changes? 

Ok. I acted on these comments in 5c6a525

strk commented:
Vicky I'm done with your suggested changes, please go on with your own additions, and let me know when done. Thanks !

strk commented:
I merged my updates so you can directly edit the docs @vlindsay (although it's generally better to create a pull-request for review if not else by CI bots)
